build-system/pr-check.js

```diff
@@ -97,6 +98,13 @@ function timedExecOrDie(cmd) {
  * Prints a summary of files changed by, and commits included in the PR.
  */
 function printChangeSummary() {
+  if (process.env.TRAVIS) {
+    console.log('Travis', colors.green('origin/master'), 'baseline:',
+        colors.cyan(gitTravisMasterBaseline()));
+  }
```

Comment:
Reformatted this block be more human readable and consistent with the formatting of this file:
```
  if (process.env.TRAVIS) {
    console.log(fileLogPrefix, colors.cyan('origin/master'), 'is currently at commit',
        colors.cyan(gitTravisMasterBaseline()));
  }
```

Reply:
Done